lib/acmesmith/post_issueing_hooks/shell.rb

```diff
@@ -0,0 +1,23 @@
+require 'erb'
+require 'acmesmith/post_issueing_hooks/base'
+
+module Acmesmith
+  module PostIssueingHooks
+    class Shell < Base
+      class HostedZoneNotFound < StandardError; end
+      class AmbiguousHostedZones < StandardError; end
+
+      def initialize(common_name:, command:)
+        @common_name = common_name
+        @command = command
+      end
+
+      def execute()
+        parsed_command = ERB.new(@command).result(binding)
```

Comment:
Since it's executed via shell, user can write `"${COMMON_NAME}-will-be-removed"` here (assuming we pass common_name as `$COMMON_NAME`)